eQuiz/modules/eQuiz.Web/Areas/Admin/Views/web.config

```diff
@@ -29,7 +29,7 @@
   <system.webServer>
     <handlers>
       <remove name="BlockViewHandler"/>
-      <add name="BlockViewHandler" path="*" verb="*" preCondition="integratedMode" type="System.Web.HttpNotFoundHandler" />
+      <add name="BlockViewHandler" path="*cshtml" verb="*" preCondition="integratedMode" type="System.Web.HttpNotFoundHandler" />
```

Comment:
Do not use the Views folder for AngularJS templates.
Put the templates into the Scripts folder instead (https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#application-structure-lift-principle).